setup.cfg

```diff
@@ -30,6 +30,8 @@ exclude_lines =
 [flake8]
 max-line-length = 100
 ignore = W503, E203, E501, D100, D101, D102, D103, D104, D105, D106, D107
+per-file-ignores =
```

Comment:
can these be ignored for the specific lines you're targeting rather than the whole file please?

Reply:
In case it was not clear, that would mean adding an ignore comment to every one of these lines https://github.com/reddit/baseplate.py/blob/34cd60deb263b2da5167eab29d527b60f5ff1870/baseplate/sidecars/event_publisher.py#L12-L42

Plus any time we import things in the future in these files, it will also need to be added to them.

So at that point it seems better to me to just ignore the rules for the file. If you'd like I could mention in the comment at the top that the two linting rules are ignored and that setting can be found in setup.cfg

Reply:
And unfortunately it sounds like there's no way in-file to just mark a block to be ignored https://stackoverflow.com/questions/64428794/flake8-disable-linter-only-for-a-block-of-code

Reply:
Ah, not my favorite but not worth addressing differently. Thanks for looking into it.